README.md

```diff
@@ -1,23 +1,3 @@
-<img src="/app/assets/images/cvlogo.svg" alt="The CircuitVerse logo" width="736"/> </br></br>
-[![Financial Contributors on Open Collective](https://opencollective.com/CircuitVerse/all/badge.svg?label=Financial+Contributors&style=for-the-badge&logo=open+collective)](https://opencollective.com/CircuitVerse) 
-[![CircleCI Status](https://img.shields.io/circleci/build/github/CircuitVerse/CircuitVerse/master?label=circleci&style=for-the-badge&logo=circleci)](https://circleci.com/gh/CircuitVerse/CircuitVerse)
-[![Coveralls Coverage Status](https://img.shields.io/coveralls/github/CircuitVerse/CircuitVerse/master?label=coveralls&style=for-the-badge&logo=coveralls)](https://coveralls.io/github/CircuitVerse/CircuitVerse?branch=master)
------
-[CircuitVerse](https://circuitverse.org) is a free, open-source platform which allows users to construct digital logic circuits online. We also offer the [Interactive Book](https://learn.circuitverse.org) which teaches users on the fundamentals of modern, digital circuits. Please also see our [documentation](https://docs.circuitverse.org) or [GitHub Wiki](https://github.com/CircuitVerse/CircuitVerse/wiki/).
+Fixed issue[#5165](https://github.com/CircuitVerse/CircuitVerse/issues/5165)
 
-## Community
-We would love to hear from you! We communicate on Slack:
-
-[![Slack](https://img.shields.io/badge/chat-on_slack-purple.svg?style=for-the-badge&logo=slack)](https://circuitverse.org/slack)
-
-## Code of Conduct
-We follow the [Code of Conduct](code-of-conduct.md) of the [CircuitVerse](https://circuitverse.org) Community.
-
-## Setup
-See [`SETUP.md`](SETUP.md) for more information on setting up CircuitVerse.
-
-## Contributing
-See [`CONTRIBUTING.md`](CONTRIBUTING.md) for more information on contributing to CircuitVerse.
-
-## License
-This project is licensed under the [MIT License](LICENSE).
+Fixed wierd cropping of profile pictures and truncated the multi-line username which messes up with the layout of the search result elements
```

Comment:
Why are you changing Readme file ?

Reply:
Apologies, I thought readme file was to be changed as well.

Reply:
It's fixed now.